meta-intel-edison-bsp/recipes-bsp/u-boot/u-boot_2019.04.bb

```diff
@@ -4,3 +4,16 @@ require u-boot-target-env.inc
 require u-boot-osip.inc
 
 DEPENDS += "bc-native dtc-native acpica-native bison-native"
+
+do_configure () {
+    if [ -z "${UBOOT_CONFIG}" ]; then
+        if [ -n "${UBOOT_MACHINE}" ]; then
+            oe_runmake -C ${S} O=${B} ${UBOOT_MACHINE}
+        else
+            oe_runmake -C ${S} O=${B} oldconfig
+        fi
+	echo ${PWD}
+        ${S}/scripts/kconfig/merge_config.sh -m .config ${@" ".join(find_cfgs(d))}
```

Comment:
Send a patch with this line to ml?

Reply:
I think we can definitely send it to oe-core ml. It's quite weird actually that nobody has complained or filed a bug about this.

Reply:
Ok I'll do that.